templates/common/config/00-config.conf

```diff
@@ -46,6 +46,8 @@ username={{ .ServiceUser }}
 password = {{ .ServicePassword }}
 {{ if (index . "MemcachedServersWithInet") }}
 memcached_servers = {{ .MemcachedServersWithInet }}
+memcache_pool_dead_retry = 10
```

Comment:
Eventually, it would be great using annotations to tweak these kind of parameters, but we'll get there at some point.